docs/azure/ai/get-started-azure-openai.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Get started with the .NET and Azure Open AI
+description: Get started on how to use the Azure OpenAI with a gpt-35-turbo model, from a simple .NET console application. Get a hiking recommendation from the AI model. It consists of a simple console application, running locally, that will send request to an Azure OpenAI service deployed in your Azure subscription.
+ms.date: 02/20/2024
+ms.topic: get-started
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: fboucher
+ms.author: frbouche
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn from the sample code.
+---
+
+This sample demonstrates how to use the Azure OpenAI with a `gpt-35-turbo` model, from a simple .NET 8.0 console application. It consists of a simple console application, running locally, that will send request to an Azure OpenAI service deployed in your Azure subscription. Everything will be deployed automatically using the Azure Developer CLI.
+
+## Requirements
+
+- .NET 8.0 SDK - [Install the .NET 8.0 SDK](https://dotnet.microsoft.com/download/dotnet/8.0)
+- An Azure subscription - [Create one for free](https://azure.microsoft.com/free/)
+- Azure Developer CLI - [Install or update the Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
+
+## Getting Started
+
+Ensure that you follow the prerequisites to have access to Azure OpenAI Service as well as the Azure Developer CLI, and then follow the following guide to set started with the sample application.
+
+1. Clone/ Download the repository: [dotnet/ai-samples](https://github.com/dotnet/ai-samples)
+1. From a terminal or command prompt, navigate to the `01-HikerAI` directory.
+1. Create the Azure resources using the Azure Developer CLI:
+	```bash
+	azd up
+	```
+1. It's now time to try the console application. Depending on your Azure subscription it's possible that a few (~5) minutes more minute are required before the model deployed in Azure OpenAI get available. If you get an error message about this, wait a few minutes and try again.
+	```bash
+	dotnet run
+	```
+
+## How it works
+
```

Comment:
```suggestion
The entire application is contained within the **Program.cs** file. The first several lines of code loads up secrets and configuration values that were set in the `dotnet user-secrets` for you during the application provisioning.

    ```csharp
    // == Retrieve the local secrets saved during the Azure deployment ==========
    var config = new ConfigurationBuilder().AddUserSecrets<Program>().Build();
    string openAIEndpoint = config["AZURE_OPENAI_ENDPOINT"];
    string openAIDeploymentName = config["AZURE_OPENAI_GPT_NAME"];
    string openAiKey = config["AZURE_OPENAI_KEY"];
    
    // == Creating the AIClient ==========
    var endpoint = new Uri(openAIEndpoint);
    var credentials = new AzureKeyCredential(openAiKey);
    ```

```